tests/lib/rules/jsx-no-useless-fragment.js

```diff
@@ -72,6 +76,12 @@ ruleTester.run('jsx-no-useless-fragment', rule, {
       errors: [{messageId: 'NeedsMoreChidren', type: 'JSXFragment'}],
       parser: parsers.BABEL_ESLINT
     },
+    {
+      code: '<>{}</>',
```

Comment:
this seems like it's just a syntax error, and wouldn't parse in the first place?

Reply:
It's totally legal. CodeSandbox here:

https://codesandbox.io/s/jsx-no-useless-fragment-qw6cj

In this case the `{}` is useless, so fixing it into `<></>` could be nice.

Reply:
wow, ok then :-)